client/src/Tools/FetchData/fetchContestsfunctions.js

```diff
@@ -0,0 +1,21 @@
+import { requestOptionsGET } from './requestOptions';
+
+export async function getContestsCards(taker) {
+  let url = '/api/contests/card/data';
+  if (taker) {
+    url = `/api/contests/card/data?taker=${taker}`;
+  }
+
+  try {
+    const response = await fetch(url, requestOptionsGET);
+    if (response.status === 200) {
+      const result = await response.json();
+      return result;
+    } else {
+      return null;
```

Comment:
Is there a check for this situation on the frontend? You could also use the early return pattern

Reply:
yes, till now if returns null, then text "Nie ma zawodów" will be rendered instead of contest card